src/meta/size.rs

```diff
@@ -15,49 +15,50 @@ pub enum Unit {
 pub struct Size {
     value: u64,
     unit: Unit,
+    bytes: u64,
 }
 
 impl<'a> From<&'a Metadata> for Size {
     fn from(meta: &Metadata) -> Self {
         let len = meta.len();
-
-        if meta.is_file() {
```

Comment:
Is there a reason why you removed this check?

Reply:
it didnt' return a size for directories. and therefore it produced some weird result.
directories have a size, depending how many files are in them.
returning the size, produced the same result as one know's from `ls`